scripts/relocate-doc-in-xsd.xsl

```diff
@@ -9,7 +9,7 @@
     <xsl:template match="xs:documentation">
         <xs:documentation>
             <xsl:text disable-output-escaping="yes">&lt;![CDATA[</xsl:text>
-            <xsl:copy-of select="*"/>
+            <xsl:copy-of select="*|text()"/>
```

Comment:
what does this change?

Reply:
This: 
![image](https://github.com/4Soft-de/harness-model/assets/56342691/e27ddbec-dcc9-4e91-b3fd-f1de99dec98c)
The old version dropped embedded documentation that was just plain text and not html formatted like this one:

```xml
<xs:complexType name="AbstractLocalizedString" abstract="true">
   <xs:annotation>
       <xs:documentation xml:lang="en">Abstract super-class for Localized text values.</xs:documentation>
   </xs:annotation>
....
<xs:sequence>
```